e2e_samples/fabric_dataops_sample/scripts/cleanup_infrastructure.sh

```diff
@@ -61,6 +61,7 @@ fabric_bearer_token=""
 fabric_api_endpoint="https://api.fabric.microsoft.com/v1"
 
 # Fabric related variables
+adls_gen2_connection_name="conn-adls-gen2-$base_name"
```

Comment:
it will be good if the name refers to storage account name.  

adls_gen2_connection_name="conn-adls-gen2-st${base_name}"

Reply:
added `st` to the connection name